src/utils/shared.py

```diff
@@ -69,3 +69,10 @@ def yes_or_no(question):
             return True
         if reply[0] == 'n':
             return False
+
+
+def day_month(date):
+    return '{}-{}'.format(
+        date.day,
+        date.month,
```

Comment:
Should probably use `.strftime` instead:
`return date.strftime("%d-%m")`

Also, worth rendering month names for the benefit of our fellow American friends?
`return date.strftime("%d-%B")`